docker-test-setup-api/02-init-chain.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+source ./common.sh
+
+$DC stop blockchain
+$DC rm -f blockchain
+$DC stop chain-{0..2}-validator chain-seed
+$DC rm -f chain-{0..2}-validator chain-seed
+
+${BB} rm -rf data/chain-{0..2}-validator data/chain-seed
+${BB} mkdir -p data/chain-{0..2}-validator/config data/chain-seed/config
+${BB} chmod -R a+rwX data/chain-{0..2}-validator/config data/chain-seed/config
+${BB} rm -rf data/deployments
+
+# has blockchain as dependency
+$DC up -d blockchain
+sleep 5
+$DC up deploy-contracts
+
+bash get-contracts.sh
+# setup chain-seed
+$DC run --rm --no-deps chain-seed init \
+    --root /chain \
+    --blocktime 1 \
+    --listen-address tcp://0.0.0.0:${TM_RPC_PORT} \
+    --role seed
+
+seed_node=$(cat data/chain-seed/config/node_key.json.id)@chain-seed:${TM_P2P_PORT}
+
+${BB} sed -i "/^moniker/c\moniker = \"chain-seed\"" data/chain-seed/config/config.toml
+
+# configure validators and keypers 0-2
+for num in {0..2}; do
+    validator_cmd=chain-$num-validator
+
+    $DC run --rm --no-deps ${validator_cmd} init \
+        --root /chain \
+        --genesis-keyper 0x440Dc6F164e9241F04d282215ceF2780cd0B755e \
+        --blocktime 1 \
+        --listen-address tcp://0.0.0.0:${TM_RPC_PORT} \
+        --role validator
+
+    validator_id=$(cat data/${validator_cmd}/config/node_key.json.id)
+    validator_node=${validator_id}@${validator_cmd}:${TM_P2P_PORT}
+    validator_config_path=data/${validator_cmd}/config/config.toml
+
+    # share genesis
+    if [ $num -eq 0 ]; then
+        for destination in data/chain-seed/config/ data/chain-{1..2}-validator/config/; do
```

Comment:
I know, this is a copy and past from elsewhere, but I think this could just be:

```suggestion
        for destination in data/chain-seed/config/ data/chain-${num}-validator/config/; do
```
or it could be pulled out of the `for num in {0..2}; do`-loop and just be
```
for destination in data/chain-seed/config/ data/chain-{1..2}-validator/config/; do
  ${BB} cp -v data/chain-0-validator/config/genesis.json "${destination}"
done
```